travis-ci/build_steps.sh

```diff
@@ -98,11 +101,16 @@ function do_build_lib {
             local bitness=64
             local target_flags="TARGET=ARMV8"
             ;;
+        arm64)
+            local bitness=64
+            local target_flags="TARGET=VORTEX"
+            ;;
```

Comment:
Could you add comments here that `aarch64` is for ARMv8, `arm64` is for Apple silicon? I find it confusing since the two terms are commonly used to describe ARMv8.

Reply:
Maybe the `arm64` entry could just be renamed to `vortex` or `arm64vortex`.

Reply:
maybe `arm64_apple` and `aarch64_arm`

Reply:
plat is taken from distutils, so we can't rename it

Reply:
Indeed. Then I think a comment would help understand what's going on. The VORTEX target for OpenBLAS is Apple specific but the ARM name is not so this config is not necessarily easy to understand for people new to the matter.